### ISSUE TITLE
Hot Keys for fast switching between Timeline and Replys

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#42]
```
It would be cool to add this feature :)
```

Original issue reported on code.google.com by `mich.ba...@gmail.com` on 13 Jan 2010 at 9:14

**Comment by GoogleCodeExporter** [comment]
Original comment by `tro...@gmail.com` on 14 Jan 2010 at 10:51
- Changed state: **Accepted**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Closed by 3dda98f15c revision.

Ctrl+1/2/3
```

Original comment by `tro...@gmail.com` on 29 Jan 2010 at 5:05
- Changed state: **Fixed**
